src/OrchardCore.Modules/OrchardCore.Navigation/Manifest.cs

```diff
@@ -5,7 +5,7 @@
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Navigation.",
+    Description = "The Navigation module allows to define menus and display them.",
```

Comment:
Or perhaps: "The Navigation module allows **you** to define menus and display them."?

Reply:
ok